### ISSUE TITLE
Missing ID Field on Place Object Despite Being Listed in PlaceField Enum

### ISSUE THREAD

**Issue by AristideVB** [matanshukry/flutter_google_places_sdk#61]
While using the `flutter_google_places_sdk` plugin, I noticed that the `PlaceField` enum lists an `Id` option. However, the `Place` object doesn't seem to have a corresponding `ID` field.

#### Code Snippets

The `PlaceField` enum includes an `Id`:

```dart
enum PlaceField {
  Address,
  AddressComponents,
  BusinessStatus,
  Id,
  @JsonValue('LAT_LNG') Location,
  // ... rest of the fields
}
```

The `Place` object definition doesn't have a field for `ID`:

```dart
@freezed
class Place with _$Place {
  const factory Place({
    required String? address,
    required List<AddressComponent>? addressComponents,
    // ... rest of the fields
  }) = _Place;
}
```

#### Suggested Solution

To make the API consistent, it would be beneficial to include an `ID` field in the `Place` class. This would align with the options available in `PlaceField` enum.

**Comment by kyadalu1** [comment]
https://www.youtube.com/watch?v=3M5fKhc1PJw

**Comment by matanshukry** [comment]
@kyadalu1 Implemented and published under sdk version 0.3.7

**Comment by AristideVB** [comment]
Thank you @matanshukry  🙏
